classes/help.php

```diff
@@ -155,32 +154,32 @@ function print_page() {
 
 												<tr valign="top">
 													<td width="150">
-														<strong><?php _e( 'Starting Location', 'SimpleMap' ); // XSS ok. ?></strong>
+														<strong><?php _e( 'Starting Location', 'simplemap' ); // XSS ok. ?></strong>
 													</td>
-													<td><?php _e( 'Enter the location the map should open to by default, when no location has been searched for. If you do not know the latitude and longitude of your starting location, enter the address in the provided text field and press "Geocode Address."', 'SimpleMap' ); // XSS ok. ?></td>
+													<td><?php _e( 'Enter the location the map should open to by default, when no location has been searched for. If you do not know the latitude and longitude of your starting location, enter the address in the provided text field and press "Geocode Address."', 'simplemap' ); // XSS ok. ?></td>
 												</tr>
 
 												<tr valign="top">
 													<td width="150">
-														<strong><?php _e( 'Auto-Load Database', 'SimpleMap' ); // XSS ok. ?></strong>
+														<strong><?php _e( 'Auto-Load Database', 'simplemap' ); // XSS ok. ?></strong>
 													</td>
 													<td>
-														<?php printf( __( '%s No auto-load:%s Locations will not load automatically.', 'SimpleMap' ), '<strong>', '</strong>' ); ?>
+														<?php printf( __( '%s No auto-load:%s Locations will not load automatically.', 'simplemap' ), '<strong>', '</strong>' ); ?>
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found
* Line exceeds 120 characters; contains 179 characters

<!-- Comment by [SideCI](https://sideci.com) -->